src/main/java/org/wikivoyage/listings/output/Xml.java

```diff
@@ -0,0 +1,63 @@
+package org.wikivoyage.listings.output;
+
+import org.wikivoyage.listings.entity.WikivoyagePOI;
+import org.wikivoyage.listings.utils.XMLSimpleNode;
+import org.wikivoyage.listings.utils.XMLSimpleNodeException;
+
+public class Xml extends NavigationXMLOutputFormat {
+    private static Integer nodeId = 0;
+    private boolean userDefined;
+
+    public XMLSimpleNode createXml(WikivoyagePOI[] pois) throws XMLSimpleNodeException
+    {
+        XMLSimpleNode genericNode = new XMLSimpleNode("generic")
+                .attrib("version", "0.5")
+                .attrib("generator", "wikivoyage-pois-converter");
+
+        for (WikivoyagePOI poi: pois) {
+            XMLSimpleNode poiNode = new XMLSimpleNode(genericNode, "node")
+                    .attrib("id", nodeId.toString())
+                    .attrib("visible", "true")
```

Comment:
hmm, this was in OsmXml.java, the "visible" attribute is only for Osm? If so, I can strip this also off